### PR TITLE
world spec rebased

### PR DESCRIPTION
**Pull request by suisha**
rebases https://github.com/ipfs/webui/pull/253

### CODE REVIEW

File: test/components/world-map.spec.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('WorldMap', () => {
+  const c = [[12, 14]]
+
+  it('should render an svg', () => {
+    const el = render(<WorldMap coordinates={c}/>)
+    expect(el.find('svg').length).to.equal(1)
+  })
+
+  it('should have the correct coordinates', () => {
```
Comment (suisha) [inline test/components/world-map.spec.js]: fixed typo

**Comment by dignifiedquire** [comment]
Thanks. For the future you can use `git push -f <remote> <branch>` to overwrite a branch so you don't need to make a new PR. Also looks like there are some conflicts with the work I did this morning, could you try and resolve these please?

**Comment by suisha** [comment]
Thanks for the tip. :+1:

**Comment by dignifiedquire** [comment]
Thanks for the work :) :octocat:

**Pull request merged**